checks/raw/security_policy.go

```diff
@@ -61,7 +61,13 @@ func SecurityPolicy(c *checker.CheckRequest) (checker.SecurityPolicyData, error)
 		return checker.SecurityPolicyData{}, err
 	}
 
-	// Return raw results.
+	// Modify the path to include org name and return raw results.
+	for i := range files {
```

Comment:
instead of patching the results, it may be cleaner to update isSecurityFile function to generate the right data. Wdut?

Reply:
Modifying the func ```isSecurityPolicyFile``` including its input params and its call chain was my first idea, but my concerns are:

1. if we would like to include the orgname checking along with the filename checking into this func, at least one extra param is needed. Since the type of this func is `` fileparser.DoWhileTrueOnFilename``` which doesn't include the client contexts (especially the needed org name). AFAIK, in golang, what I can do is to create a new type like ```fileparser.DoWhileTrueOnFilenameWithClientContexts``` since I find golang doesn't support neither optional parameters nor method overloading. And if I do this, it seems the following modifications along with this would be much more.

2. Maybe we should keep the atomicity of this func? That is, only to check the filename itself. Otherwise,  it would be a little confusing for checking both ```filename``` and ```orgname + hostname + filename in this func.

3. If the information is only used for the logger, in my view the policy name results patching would be enough (and clean).

wdut? @laurentsimon

Reply:
I'll look into it and see which solution is better.